### PR TITLE
Create TextTypingGame

### PR DESCRIPTION
**Pull request by briankarlsayen**
new entry Text Typing Game

### CODE REVIEW

**Comment by galacemiguel** [comment]
Hi, @briankarlsayen! It looks like there are some formatting issues with your entry. 🧐

Can you run the following and push again? Thanks!

```
yarn prettier --write src/entries/TextTypingGame
```

**Comment by briankarlsayen** [comment]
> Hi, @briankarlsayen! It looks like there are some formatting issues with your entry. 🧐
> 
> Can you run the following and push again? Thanks!
> 
> ```
> yarn prettier --write src/entries/TextTypingGame
> ```

ok na boss

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
     "react-spinners": "^0.11.0",
+    "rehackt-challenge-week-one": "https://github.com/briankarlsayen/rehackt-challenge-week-one.git",
```
Comment (galacemiguel) [inline package.json]:
```suggestion
```

File: yarn.lock [hunk 2]
```diff
@@ -9901,6 +9901,29 @@ regjsparser@^0.7.0:
...
+    prop-types "^15.7.2"
+    react "^17.0.2"
+    react-audio-player "^0.17.0"
+    react-dom "^17.0.2"
+    react-router-dom "^5.3.0"
+    react-scripts "4.0.3"
+    react-spinners "^0.11.0"
+    styled-components "^5.3.1"
+    web-vitals "^1.0.1"
+
```
Comment (galacemiguel) [inline yarn.lock]:
```suggestion
```

File: yarn.lock [hunk 3]
```diff
@@ -11648,8 +11671,10 @@ watchpack@^1.7.4:
   resolved "https://registry.yarnpkg.com/watchpack/-/watchpack-1.7.5.tgz#1267e6c55e0b9b5be44c2023aed5437a2c26c453"
   integrity sha512-9P3MWk6SrKjHsGkLT2KHXdQ/9SNkyoJbabxnKOoJepsvJjJG8uYTR3yTPxPQvNDI3w4Nz1xnE0TLHK4RIVe/MQ==
   dependencies:
+    chokidar "^3.4.1"
     graceful-fs "^4.1.2"
     neo-async "^2.5.0"
+    watchpack-chokidar2 "^2.0.1"
```
Comment (galacemiguel) [inline yarn.lock]:
```suggestion
    graceful-fs "^4.1.2"
    neo-async "^2.5.0"
```

**Review by galacemiguel — CHANGES_REQUESTED** [review]
One last thing! Pa-undo na lang ng changes mo doon sa `package.json` at `yarn.lock`. 'Di na dapat siya kailangan. 😄

Ganda ng gawa mo though! Maybe my favorite entry so far.

**Comment by briankarlsayen** [comment]
Sir pa check nlng po ulet kung may conflict pa. Thanks.

File: yarn.lock (hunk 3 above)
Comment (galacemiguel) [reply yarn.lock]: Ito na lang kulang, @briankarlsayen. 🙏

Comment (briankarlsayen) [reply yarn.lock]: yarn sir @galacemiguel ok na ata haha

**Comment by galacemiguel** [comment]
> Sir pa check nlng po ulet kung may conflict pa. Thanks.

@briankarlsayen Sumobra ang na-delete mo sir. 😅

Dapat walang diff ang [`yarn.lock` sa **Files changed**](https://github.com/reactph/rehackt-challenge-week-one/pull/16/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de
):

image

**Comment by briankarlsayen** [comment]
@galacemiguel idedelete ko lng ung     

- graceful-fs "^4.1.2"
- neo-async "^2.5.0"

or 

    chokidar "^3.4.1"
    graceful-fs "^4.1.2"
    neo-async "^2.5.0"
    watchpack-chokidar2 "^2.0.1"

**Review by galacemiguel — APPROVED** [review]
Ito lang @briankarlsayen para mabalik sa dati yung `yarn.lock`. 😄 

```
git checkout main -- yarn.lock
```

Pinush ko na. Thanks for contributing and we hope to see you at our event later! ✨

**Pull request merged**